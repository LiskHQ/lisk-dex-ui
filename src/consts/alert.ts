export enum AlertVariant {
  success = 'success',
  info = 'info',
}