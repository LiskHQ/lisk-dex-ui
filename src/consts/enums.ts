export enum ProposalType {
  PoolIncentivization = "POOL_INCENTIVIZATION",
  Universal = "UNIVERSAL",
}