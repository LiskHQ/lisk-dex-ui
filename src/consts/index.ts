export * from './menu';
export * from './paths';
export * from './proposal';
export * from './alert';
export * from './transaction';
