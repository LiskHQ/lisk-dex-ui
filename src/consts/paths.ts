export const PATHS = {
  SWAP: '/swap',
  POOL: '/pool',
  INFO: '/info',
  GOVERNANCE: '/governance',
  CREATE_PROPOSAL: '/governance/create',
  PROPOSAL: '/governance/proposal',
};