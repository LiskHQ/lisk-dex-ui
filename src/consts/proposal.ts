export enum ProposalType {
  PoolIncentivization = "Pool Incentivization proposal",
  Universal = "Universal proposal",
}

export enum ProposalStatus {
  All_Proposals = "All Proposals",
  Active = "Active",
  Accepted = "Accepted",
  Failed = "Failed",
  Failed_Quorum = "Failed Quorum",
}