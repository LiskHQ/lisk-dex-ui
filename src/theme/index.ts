export { default as theme } from "./theme";
export { default as variables } from "./variables";
