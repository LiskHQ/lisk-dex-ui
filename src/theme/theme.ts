import {
  createTheme,
  responsiveFontSizes,
  ThemeOptions,
} from "@mui/material/styles";

import variables from "./variables";

const options: ThemeOptions = {
  palette: {
    primary: variables.primary,
    secondary: variables.secondary,
  },
  typography: {
    h1: {
      fontSize: "2.5rem",
      fontWeight: 600,
    },
    h2: {
      fontSize: "2rem",
      fontWeight: 400,
    },
    h3: {
      fontSize: "1.75rem",
      fontWeight: 400,
    },
    h4: {
      fontSize: "1.25rem",
      fontWeight: 400,
    },
    h5: {
      fontSize: "1rem",
      fontWeight: 400,
    },
    h6: {
      fontSize: "0.75rem",
      fontWeight: 400,
    },
  },
  breakpoints: {
    values: {
      xs: 0,
      sm: 576,
      md: 768,
      lg: 1024,
      xl: 1440,
      xxl: 1950,
      xxxl: 2300,
    },
  },
};

const theme = responsiveFontSizes(createTheme(options));
export default theme;
