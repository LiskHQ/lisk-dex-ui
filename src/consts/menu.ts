import { PATHS } from './paths';

export const menuItems = [
  {
    title: 'Swap',
    href: PATHS.SWAP,
  },
  {
    title: 'Pool',
    href: PATHS.POOL,
  },
  {
    title: 'Info',
    href: PATHS.INFO,
  },
  {
    title: 'Governance',
    href: PATHS.GOVERNANCE,
  },
];

export enum ThemeType {
  Light = 'Light',
  Dark = 'Dark',
}

export const currencySymbols: {
  [key: string]: string,
} = {
  USD: '$',
  EUR: 'â‚¬',
};