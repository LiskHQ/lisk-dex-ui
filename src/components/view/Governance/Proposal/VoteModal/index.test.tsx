import React from "react";
import { ThemeProvider } from "@mui/material";
import { fireEvent, render } from "@testing-library/react";
import { VoteModal, IVoteModalProps } from "./index";
import { lightTheme } from "styles/theme";
import { VoteType } from "consts";

function renderComponent(props: IVoteModalProps) {
  return render(
    <ThemeProvider theme={lightTheme}>
      <VoteModal {...props} />
    </ThemeProvider>
  );
}

describe("VoteModal", () => {
  const mockProps: IVoteModalProps = {
    openTransactionApproval: false,
    type: VoteType.Yes,
    onClose: jest.fn(),
    onVote: jest.fn(),
  }

  it("checks if the component matches the snapshot", () => {
    const { container } = renderComponent(mockProps);
    expect(container).toMatchSnapshot();
  });

  it("check close button work", () => {
    const { getByTestId } = renderComponent(mockProps);
    fireEvent.click(getByTestId("vote-modal-close-button-test"));

    expect(mockProps.onClose).toBeCalled();
  });

  it("revote work", () => {
    const { getByText } = renderComponent(mockProps);

    expect(getByText("Recast your vote")).toBeInTheDocument();
  });
});
