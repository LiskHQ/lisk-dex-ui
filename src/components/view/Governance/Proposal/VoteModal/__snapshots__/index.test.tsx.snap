// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VoteModal checks if the component matches the snapshot 1`] = `
.emotion-0 .vote-modal-background {
  position: fixed;
  left: 0;
  top: 0;
  width: 100%;
  height: 100%;
  background: rgba(0, 0, 0, 0.5);
  z-index: 10;
}

.emotion-0 .vote-modal-container {
  position: fixed;
  left: 50%;
  top: 50%;
  margin-bottom: 10rem;
  -webkit-transform: translate(-50%, -50%);
  -moz-transform: translate(-50%, -50%);
  -ms-transform: translate(-50%, -50%);
  transform: translate(-50%, -50%);
  width: 25.625rem;
  border-radius: 0.5rem;
  background: #FFFFFF!important;
  box-shadow: 0px 1px 7px rgba(0, 0, 0, 0.1);
  z-index: 10;
}

@media (max-width:575.95px) {
  .emotion-0 .vote-modal-container {
    width: calc(100% - 2rem);
    left: 1rem;
    -webkit-transform: translateY(-50%);
    -moz-transform: translateY(-50%);
    -ms-transform: translateY(-50%);
    transform: translateY(-50%);
  }
}

.emotion-0 .vote-modal-container .vote-modal-header {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  padding: 1.5rem;
  border-bottom: 0.5px solid #E6E6E6;
  color: #282828;
}

.emotion-0 .vote-modal-container .vote-modal-body {
  padding: 1rem;
}

.emotion-0 .vote-modal-container .vote-modal-body .vote-modal-voting-power {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
  margin-bottom: 2rem;
}

.emotion-0 .vote-modal-container .vote-modal-body .vote-modal-voting-power .vote-modal-voting-power-dex {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-0 .vote-modal-container .vote-modal-body .vote-modal-voting-power .vote-modal-voting-power-dex svg {
  fill: #6953F4;
  margin-left: 0.5rem;
}

.emotion-0 .vote-modal-container .vote-modal-footer {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 2rem 1rem 1.5rem 1rem;
}

.emotion-0 .vote-modal-container .vote-modal-footer .vote-modal-cancel {
  width: 10rem;
  margin-right: 1rem;
  background: none;
}

.emotion-0 .vote-modal-container .vote-modal-footer .vote-modal-cancel:hover P {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.emotion-0 .vote-modal-container .vote-modal-footer .vote-modal-cancel P {
  color: #535353;
}

.emotion-4 {
  margin: 0;
  font-size: 1.25rem;
  font-weight: 600;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  line-height: 1.235;
  letter-spacing: 0.00735em;
}

.emotion-7 {
  margin: 0;
  font-size: 0.875rem;
  line-height: 1.125rem;
  font-weight: 400;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  letter-spacing: 0.01071em;
}

.emotion-10 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  position: relative;
  min-width: 0;
  padding: 0;
  margin: 0;
  border: 0;
  vertical-align: top;
}

.emotion-11 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-flex-wrap: wrap;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
}

.emotion-12 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  cursor: pointer;
  vertical-align: middle;
  -webkit-tap-highlight-color: transparent;
  margin-left: -11px;
  margin-right: 16px;
}

.emotion-12.Mui-disabled {
  cursor: default;
}

.emotion-12 .MuiFormControlLabel-label.Mui-disabled {
  color: rgba(0, 0, 0, 0.38);
}

.emotion-13 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  position: relative;
  box-sizing: border-box;
  -webkit-tap-highlight-color: transparent;
  background-color: transparent;
  outline: 0;
  border: 0;
  margin: 0;
  border-radius: 0;
  padding: 0;
  cursor: pointer;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  vertical-align: middle;
  -moz-appearance: none;
  -webkit-appearance: none;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
  padding: 9px;
  border-radius: 50%;
  color: rgba(0, 0, 0, 0.6);
  color: #E6E6E6!important;
}

.emotion-13::-moz-focus-inner {
  border-style: none;
}

.emotion-13.Mui-disabled {
  pointer-events: none;
  cursor: default;
}

@media print {
  .emotion-13 {
    -webkit-print-color-adjust: exact;
    color-adjust: exact;
  }
}

.emotion-13:hover {
  background-color: rgba(25, 118, 210, 0.04);
}

@media (hover: none) {
  .emotion-13:hover {
    background-color: transparent;
  }
}

.emotion-13.Mui-checked {
  color: #1976d2;
}

.emotion-13.Mui-disabled {
  color: rgba(0, 0, 0, 0.26);
}

.emotion-13.Mui-checked {
  color: #6953F4!important;
}

.emotion-14 {
  cursor: inherit;
  position: absolute;
  opacity: 0;
  width: 100%;
  height: 100%;
  top: 0;
  left: 0;
  margin: 0;
  padding: 0;
  z-index: 1;
}

.emotion-15 {
  position: relative;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.emotion-16 {
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  width: 1em;
  height: 1em;
  display: inline-block;
  fill: currentColor;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  -webkit-transition: fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  font-size: 1.5rem;
  -webkit-transform: scale(1);
  -moz-transform: scale(1);
  -ms-transform: scale(1);
  transform: scale(1);
}

.emotion-17 {
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  width: 1em;
  height: 1em;
  display: inline-block;
  fill: currentColor;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  -webkit-transition: fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  font-size: 1.5rem;
  left: 0;
  position: absolute;
  -webkit-transform: scale(1);
  -moz-transform: scale(1);
  -ms-transform: scale(1);
  transform: scale(1);
  -webkit-transition: -webkit-transform 150ms cubic-bezier(0.0, 0, 0.2, 1) 0ms;
  transition: transform 150ms cubic-bezier(0.0, 0, 0.2, 1) 0ms;
}

.emotion-18 {
  overflow: hidden;
  pointer-events: none;
  position: absolute;
  z-index: 0;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  border-radius: inherit;
}

.emotion-19 {
  margin: 0;
  font-size: 1rem;
  line-height: 1.375rem;
  font-weight: 500;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  letter-spacing: 0.00938em;
}

.emotion-25 {
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  width: 1em;
  height: 1em;
  display: inline-block;
  fill: currentColor;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  -webkit-transition: fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  font-size: 1.5rem;
  left: 0;
  position: absolute;
  -webkit-transform: scale(0);
  -moz-transform: scale(0);
  -ms-transform: scale(0);
  transform: scale(0);
  -webkit-transition: -webkit-transform 150ms cubic-bezier(0.4, 0, 1, 1) 0ms;
  transition: transform 150ms cubic-bezier(0.4, 0, 1, 1) 0ms;
}

.emotion-37 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  position: relative;
  box-sizing: border-box;
  -webkit-tap-highlight-color: transparent;
  background-color: transparent;
  outline: 0;
  border: 0;
  margin: 0;
  border-radius: 0;
  padding: 0;
  cursor: pointer;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  vertical-align: middle;
  -moz-appearance: none;
  -webkit-appearance: none;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 500;
  font-size: 0.875rem;
  line-height: 1.75;
  letter-spacing: 0.02857em;
  text-transform: uppercase;
  min-width: 64px;
  padding: 6px 8px;
  border-radius: 4px;
  -webkit-transition: background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  color: #1976d2;
  width: 100%;
  background: #6953F4;
  color: #FFFFFF;
  padding: 0.75rem;
  text-transform: capitalize;
}

.emotion-37::-moz-focus-inner {
  border-style: none;
}

.emotion-37.Mui-disabled {
  pointer-events: none;
  cursor: default;
}

@media print {
  .emotion-37 {
    -webkit-print-color-adjust: exact;
    color-adjust: exact;
  }
}

.emotion-37:hover {
  -webkit-text-decoration: none;
  text-decoration: none;
  background-color: rgba(25, 118, 210, 0.04);
}

@media (hover: none) {
  .emotion-37:hover {
    background-color: transparent;
  }
}

.emotion-37.Mui-disabled {
  color: rgba(0, 0, 0, 0.26);
}

.emotion-37:hover {
  background: #A196F6;
}

.emotion-37.Mui-disabled {
  color: #FFFFFF;
  opacity: 0.7;
}

<div>
  <div
    class="emotion-0"
    data-testid="vote-modal-test"
  >
    <div
      class="vote-modal-background MuiBox-root emotion-1"
    />
    <div
      class="vote-modal-container MuiBox-root emotion-1"
    >
      <div
        class="vote-modal-header MuiBox-root emotion-1"
      >
        <h4
          class="MuiTypography-root MuiTypography-h4 emotion-4"
        >
          Recast your vote
        </h4>
      </div>
      <div
        class="vote-modal-body MuiBox-root emotion-1"
      >
        <div
          class="vote-modal-voting-power MuiBox-root emotion-1"
        >
          <p
            class="MuiTypography-root MuiTypography-body2 emotion-7"
          >
            Your voting power:
          </p>
          <div
            class="vote-modal-voting-power-dex MuiBox-root emotion-1"
          >
            <svg
              fill="#161133"
              height="20"
              viewBox="0 0 20 20"
              width="20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M8.28739 17.9646L10.3754 15.6185C10.4106 15.5833 10.3871 15.5247 10.3284 15.5247H8.02933C8.0176 15.5247 7.99413 15.513 7.9824 15.5012L6.03519 13.3194C6.02346 13.2959 6.01173 13.2725 6.02346 13.249L9.28446 7.65372C9.29619 7.63026 9.29619 7.6068 9.28446 7.59507L7.93548 5.27249C7.91202 5.2373 7.85337 5.2373 7.82991 5.27249L3.0088 13.554C2.99707 13.5775 2.99707 13.6009 3.0088 13.6244L6.87977 17.9763C6.8915 17.988 6.90323 17.9998 6.92669 17.9998H8.24047C8.2522 17.988 8.27566 17.9763 8.28739 17.9646Z"
              />
              <path
                d="M9.71911 2.03519L8.35841 4.35777C8.34668 4.38123 8.34668 4.40469 8.35841 4.41642L9.76603 6.83284L13.4962 13.2375C13.508 13.261 13.508 13.2845 13.4845 13.3079L11.5138 15.5132L9.41413 17.8827C9.37893 17.9179 9.4024 17.9765 9.46105 17.9765H12.6047C12.6165 17.9765 12.6399 17.9648 12.6517 17.9531L16.5226 13.6012C16.5344 13.5777 16.5461 13.5543 16.5344 13.5308L9.81295 2.03519C9.80122 1.98827 9.74257 1.98827 9.71911 2.03519Z"
              />
            </svg>
            <p
              class="MuiTypography-root MuiTypography-body2 emotion-7"
            >
              42 LSKDEX
            </p>
          </div>
        </div>
        <div
          class="MuiFormControl-root emotion-10"
        >
          <div
            aria-labelledby="demo-radio-buttons-group-label"
            class="MuiFormGroup-root emotion-11"
            role="radiogroup"
          >
            <label
              class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd emotion-12"
              data-testid="vote-modal-radio"
            >
              <span
                class="MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root MuiRadio-root MuiRadio-colorPrimary Mui-checked MuiRadio-root MuiRadio-colorPrimary emotion-13"
              >
                <input
                  checked=""
                  class="PrivateSwitchBase-input emotion-14"
                  name="radio-buttons-group"
                  type="radio"
                  value="Yes"
                />
                <span
                  class="emotion-15"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium emotion-16"
                    data-testid="RadioButtonUncheckedIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                    />
                  </svg>
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium emotion-17"
                    data-testid="RadioButtonCheckedIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                    />
                  </svg>
                </span>
                <span
                  class="MuiTouchRipple-root emotion-18"
                />
              </span>
              <span
                class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label emotion-19"
              >
                Yes
              </span>
            </label>
            <label
              class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd emotion-12"
            >
              <span
                class="MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root MuiRadio-root MuiRadio-colorPrimary MuiRadio-root MuiRadio-colorPrimary emotion-13"
              >
                <input
                  class="PrivateSwitchBase-input emotion-14"
                  name="radio-buttons-group"
                  type="radio"
                  value="No"
                />
                <span
                  class="emotion-15"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium emotion-16"
                    data-testid="RadioButtonUncheckedIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                    />
                  </svg>
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium emotion-25"
                    data-testid="RadioButtonCheckedIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                    />
                  </svg>
                </span>
                <span
                  class="MuiTouchRipple-root emotion-18"
                />
              </span>
              <span
                class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label emotion-19"
              >
                No
              </span>
            </label>
            <label
              class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd emotion-12"
            >
              <span
                class="MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root MuiRadio-root MuiRadio-colorPrimary MuiRadio-root MuiRadio-colorPrimary emotion-13"
              >
                <input
                  class="PrivateSwitchBase-input emotion-14"
                  name="radio-buttons-group"
                  type="radio"
                  value="Pass"
                />
                <span
                  class="emotion-15"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium emotion-16"
                    data-testid="RadioButtonUncheckedIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                    />
                  </svg>
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium emotion-25"
                    data-testid="RadioButtonCheckedIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                    />
                  </svg>
                </span>
                <span
                  class="MuiTouchRipple-root emotion-18"
                />
              </span>
              <span
                class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label emotion-19"
              >
                Pass
              </span>
            </label>
          </div>
        </div>
      </div>
      <div
        class="vote-modal-footer MuiBox-root emotion-1"
      >
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium vote-modal-cancel emotion-37"
          data-testid="vote-modal-close-button-test"
          tabindex="0"
          type="button"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 emotion-19"
          >
            Cancel
          </p>
          <span
            class="MuiTouchRipple-root emotion-18"
          />
        </button>
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium Mui-disabled MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium vote-modal-confirm emotion-37"
          data-testid="vote-modal-button-test"
          disabled=""
          tabindex="-1"
          type="button"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 emotion-19"
          >
            Revote
          </p>
        </button>
      </div>
    </div>
  </div>
</div>
`;
